### ISSUE TITLE
Point question

### ISSUE THREAD

**Issue by JackZhang1994** [dat-ng/ar-location-based-android#17]
Why is the point so unstable? It doesn't stop moving in the screen, and then it shows off for a while, and I don't have a mobile phone. Can you answer my question? Thanks~

**Comment by dat-ng** [comment]
Please check your phone whether has compass and accelerometer sensors

2018-01-15 9:37 GMT+07:00 JackZhang1994 <notifications@github.com>:

**Comment by JackZhang1994** [comment]
![image](https://user-images.githubusercontent.com/20792223/34925864-0d6d1cb2-f9e7-11e7-8707-a7297ab96c8c.png)
Here regist a sensor listener.
![image](https://user-images.githubusercontent.com/20792223/34925886-2bd440f4-f9e7-11e7-837b-64376961a90e.png)
And it can go into this method.

**Comment by dat-ng** [comment]
What is your device?

2018-01-15 10:30 GMT+07:00 JackZhang1994 <notifications@github.com>:

**Comment by JackZhang1994** [comment]
In OnDraw(Canvas canvas) method, I set a log.
![image](https://user-images.githubusercontent.com/20792223/34926008-c2bc1ba4-f9e7-11e7-9b15-32ea5165033b.png)
And this is the result.
![image](https://user-images.githubusercontent.com/20792223/34926033-e2010fc4-f9e7-11e7-877d-3e640d6ca707.png)
A large change in coordinate points, and why?

**Comment by JackZhang1994** [comment]
My device is MEIZU M5, and another is One Plus 5.

**Comment by dat-ng** [comment]
Because I set SENSOR_DELAY_GAME flag in sensor register listener, it
probably depends on devices, but I think sensor trigger and frequently
update value for each 37-39 mili seconds

2018-01-15 10:41 GMT+07:00 JackZhang1994 <notifications@github.com>:

**Comment by JackZhang1994** [comment]
Well, what I mean is 
x:826~863
y:154~201
but I didn't move my device.

**Comment by JackZhang1994** [comment]
Ehnnn, what I mean is...
When I fix the cell phone, the value of X and Y is still changing, the range of the change is 50. What is the cause? Do you have a way to solve it?
And another question, when I set up difficult altitude, the location of the point is always in the center of the screen, just like no altitude. Can you tell me why? Thank you very much~

**Comment by ShantiRanjanDas** [comment]
Yes, I have also the similar situation. The points are moving and are not on a particular position, While I move my phone they also move, but they must be in one position.

**Comment by makisp** [comment]
For my case, i had no problem with Xiaomi Mi A1 running Nougat. 
After updating to Oreo this started happening...

**Comment by pankajjangid** [comment]
i had a same issue if anyone got the solution please help me

**Comment by devidvogel** [comment]
Hey, is there a solution for this problem? I have the same issue with a Google Pixel with Android 9.
Thanks.

**Comment by akshayshelke232** [comment]
@pankajjangid @dat-ng @JackZhang1994 @travs15 @makisp  did you find any solution to this problem?
